src/components/Table/components/RenderRowOrNestedRow/RenderRowOrNestedRow.tsx

```diff
@@ -146,7 +146,7 @@ const RenderRowOrNestedRow = <T extends { [key: string]: unknown }>({
               <table css={tableStyle()()}>
                 <tbody>
                   <RenderRowWithCells
-                    {...{ isChecked, toggleIsChecked }}
+                    {...{ isChecked, toggleChecked: toggleIsChecked }}
```

Comment:
why renaming this? `isChecked` and `toggleIsChecked` are the same thing by renaming these seem different